### PR TITLE
Clarify webhook auth token details

### PR DESCRIPTION
**Pull request by feorlen**
Update text about webhook auth tokens to clarify MinIO uses the value specified as it is. So if it needs `Bearer` or `Basic` or something else, that must be included in the string.

Affected items:
- `MINIO_LOGGER_WEBHOOK_AUTH_TOKEN`
- `MINIO_AUDIT_WEBHOOK_AUTH_TOKEN`
- `MINIO_NOTIFY_WEBHOOK_AUTH_TOKEN`
- `mc admin config set logger_webhook auth_token`
- `mc admin config set audit_webhook auth_token`
- `mc admin config set notify_webhook auth_token`

Staged:

_Publish Server or Audit Logs to an External Service_
http://192.241.195.202:9000/staging/DOCS-407/linux/operations/monitoring/minio-logging.html#publish-server-logs-to-http-webhook
http://192.241.195.202:9000/staging/DOCS-407/linux/operations/monitoring/minio-logging.html#publish-audit-logs-to-http-webhook

_mc admin config_
http://192.241.195.202:9000/staging/DOCS-407/linux/reference/minio-mc-admin/mc-admin-config.html#mc-conf.logger_webhook.auth_token
http://192.241.195.202:9000/staging/DOCS-407/linux/reference/minio-mc-admin/mc-admin-config.html#mc-conf.audit_webhook.auth_token
http://192.241.195.202:9000/staging/DOCS-407/linux/reference/minio-mc-admin/mc-admin-config.html#mc-conf.notify_webhook.auth_token

_MinIO Server_
http://192.241.195.202:9000/staging/DOCS-407/linux/reference/minio-server/minio-server.html#envvar.MINIO_LOGGER_WEBHOOK_AUTH_TOKEN
http://192.241.195.202:9000/staging/DOCS-407/linux/reference/minio-server/minio-server.html#envvar.MINIO_AUDIT_WEBHOOK_AUTH_TOKEN
http://192.241.195.202:9000/staging/DOCS-407/linux/reference/minio-server/minio-server.html#envvar.MINIO_NOTIFY_WEBHOOK_AUTH_TOKEN

Fixes https://github.com/minio/docs/issues/407

### CODE REVIEW

**Comment by feorlen** [comment]
Screenshot of rendered page, starting at "Replace TOKEN with an authentication token of the appropriate type..."

**Comment by feorlen** [comment]
Example of revised text:

Screenshot of Publish Server Logs to HTTP Webhook, beginning with "Replace TOKEN with an authentication token of the appropriate type"

File: source/includes/common-mc-admin-config.rst [hunk 1]
```diff
@@ -996,17 +996,6 @@ Specify the URL for the webhook service.
 
 .. end-minio-notify-webhook-endpoint
 
-.. start-minio-notify-webhook-auth-token
```
Comment (feorlen) [inline source/includes/common-mc-admin-config.rst:999]: Removed this include because the needed text is slightly different in the two locations it was used.
(Also why the others aren't includes.)

File: source/operations/monitoring/minio-logging.rst [hunk 2]
```diff
@@ -221,7 +301,7 @@ Audit Log Structure
 
 MinIO audit logs resemble the following JSON document:
 
-- The ``api.timeToFirstBite`` and ``api.timeToResponse`` fields are expressed
+- The ``api.timeToFirstByte`` and ``api.timeToResponse`` fields are expressed
```
Comment (feorlen) [inline source/operations/monitoring/minio-logging.rst:304]: Drive-by typo fix, thank you @kannappanr

**Review by ravindk89 — APPROVED** [review]
